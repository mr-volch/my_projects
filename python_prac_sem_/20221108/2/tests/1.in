kero
(1,2),3(4),(5,6)
(1,2,3,4,5,6)
(1,1), (1,2), (1,11)
(1,1), (4,0), (0,11)
