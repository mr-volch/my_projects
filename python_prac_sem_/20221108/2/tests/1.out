Invalid input
Invalid input
Invalid input
Not a triangle
14.50
